repository/r.json

```diff
@@ -1528,6 +1528,17 @@
 				}
 			]
 		},
+		{
+			"name": "ride",
```

Comment:
Why is it called "ride"? Is that a name that would make sense to the target audience? To me, if it is the syntax for a language called "Lingua Franka", why not call the package "Lingua Franka Syntax"?

Reply:
Hi, ride is the name of our App-Development SDK so the target audience will probably search for that. Syntax highlighting is only the first step of what the extension could provide.

I'm fine if the plugin is for now named to 'Lingua Franka Syntax' and once more features have been added renamed to 'ride'.

Reply:
But not that there's also an R-IDE package: https://packagecontrol.io/packages/R-IDE

Reply:
Could also name it 'Franka Ride', would that be ok?

Reply:
> But not that there's also an R-IDE package

I don't think that's a problem though.

> Could also name it 'Franka Ride', would that be ok?

If "Ride" makes any sense for your context that's all fine. It doesn't really matter what it's called, but for an outsider it just seemed a bit random. So, "ride" is ok 🤷🏻